### DISCUSSION TITLE
Plugin Sections

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/CostControl#6] [Ideas]
Some ideas to include as features in the plugin

- [x] Add Separate Cost Control section
- [x] Manual & Live Currency Rates
  - [x] Add Comments
- [ ] Subscriptions
  - Is it for IN or OUT payments? 
- [ ] Raw Costs
  - @salazarjoelo https://kanboard.discourse.group/t/does-anyone-associate-costs-or-budgets-with-projects-or-tasks-or-kanboard-as-a-whole/2751/2?u=aljawaid
- [ ] Petty Cash

**Comment by salazarjoelo** [comment]
Well @aljawaid I appreciate your message.
About the features you propose, all of them look great to me.

Please let me write some ideas an I will back on it tomorrow; I would like
to share with you the real cases I think these features could be included
in.

**Comment by salazarjoelo** [comment]
Below I share flows that seem applicable to any type of project:
For example, I do a project that consists of sales of any product that I
make.
Each task is an order and has the following phases (kanban columns):
Marketing (subtasks: advertising campaign, lead generation, conversion,
contact, quote, or similars), Order (Subtasks: formal customer order,
purchase order with product specifications (s) signed by both parties,
invoicing, advance payment), manufacturing (Subtasks: purchase of materials
and components, manufacturing, testing and approval, preparation of
documentation for use), Delivery of the product (s) (Subtasks: final
payment, delivery of the product, training in use, expiration of the
warranty period.

This flow is almost the same for the sale of services.

I think that each project can be a specific manufacturing or service model,
each task is a client/order and its subtasks, according to each product or
service model for sale.
The creation of new tasks can be automated. When these are created, their
subtasks are created as well; some subtasks would be the lists of materials
and components that are needed for each order of the manufactured product
or service provided, at this point is where I think a feature of your Cost
Control plugin would work very well to have the costs of each input and
component required for the product or service.

In addition, you can allow prices to be updated manually, and obviously
quoted in foreign currencies, for example, here in Mexico, it is normal to
incorporate components that are purchased in US dollars.

This is what I wanted to share with you, sorry for the delay but I had
difficult days. Receive greetings @aljawaid

**Reply by aljawaid** [reply]
cool, I will look into it... no promises.... this is a slow-developing plugin

**Comment by salazarjoelo** [comment]
Yes, I got it clear. I just shared with you all the stuff, for your
knowledge and vision.

*"Gracias por pensar en un Medio Ambiente Saludable"*
Atenta y Respetuosamente

Joel Salazar Ramírez
